README.md

```diff
@@ -1,5 +1,23 @@
 ## Woopra Android SDK
 
+## Installation
```

Comment:
imo: add a blank line between headings and follow-up text body to improve readability and also potentially conform the `markdownlint`: https://github.com/markdownlint/markdownlint